### DISCUSSION TITLE
When using docker version of naabu , The -l flag doesn't work

### DISCUSSION THREAD

**Discussion by MR-pentestGuy** [projectdiscovery/naabu#520] [Q&A] [answered]
Hi I have been running the docker image for naabu . 
The command i have given --> docker run  -it projectdiscovery/naabu -l ips.txt 
Error i get --> [FTL] Could not run enumeration: open ips.txt: no such file or directory

So now I started to use forloop for running against list of ips 

Thanks

**Comment by pnhb112345** [comment, marked as answer]
cat ips.txt | docker run -i projectdiscovery/naabu
